test/ex_doc/language/erlang_test.exs

```diff
@@ -180,7 +180,193 @@ defmodule ExDoc.Language.ErlangTest do
     end
   end
 
-  describe "autolink_doc/2 for extras" do
+  describe "autolink_doc/2 for markdown" do
+    test "module in module code", c do
+      assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
+               assert autolink_markdown("`erlang_bar`", c) ==
+                        ~s|<code class="inline">erlang_bar</code>|
+             end) == ""
```

Comment:
there's no need to assert nothing was printed. While we don't have an automatic way to fail the build if warning was emitted, it's fine for now to rely on ourselves to spot these when running tests locally. Btw don't worry about it, I'll fix these up after this PR lands.

```suggestion
    assert autolink_markdown("`erlang_bar`", c) ==
             ~s|<code class="inline">erlang_bar</code>|
```

Reply:
You are braver than I am :smile: I've stopped trusting myself to notice things manually

Reply:
We also support concurrent captures, which means that `== ""` may fail if we end-up capturing something from another test. :)